### ISSUE TITLE
[FEATURE] Color blind accessible build status indicator

### ISSUE THREAD

**Issue by vincent-paing** [vincent-paing/CCDroidX#4]
### Value Statement

As a user with color blindness, I want to know the difference between each build status so that I can actually use the app

### Summary

Currently, the indicator is just plain old red, green, yellow circles. These are not accessible as if you have color blindness, they'd all look the same. Instead, we should use tick icon for successful, cross icon for fail and play icon for running builds.

#### AC 1 : Icon Indicator

| Given | When | Then | 
| ----------- | -----------  |----------- |
| I have a project | Build is successful | The indicator should be green circle with tick icon cut out in middle |
| I have a project | Build is failed | The indicator should be red circle with cross icon cut out in middle |
| I have a project | Build is running | The indicator should be yellow circle with play icon cut out in middle |
